animeal/src/Networking/NetworkService.swift

```diff
@@ -0,0 +1,74 @@
+// System
+import UIKit
+
+// SDK
+import Services
+import Amplify
+import AWSDataStorePlugin
+import AWSAPIPlugin
+import AWSPluginsCore
+import UIComponents
+
+protocol NetworkServiceHolder {
+    var networkService: NetworkServiceProtocol { get }
+}
+
+typealias MyClientResponse<ResponseType: Decodable> = Result<ResponseType, Error>
```

Comment:
Maybe we can use Result as is, without type alias ?

Reply:
Done